packages/manager/apps/web/client/app/domain/zone/domain-zone-dns.html

```diff
@@ -25,9 +25,9 @@
                 <button
                     class="btn btn-block btn-default"
                     type="button"
-                    data-translate="domain_tab_ZONE_no_zone_activate_btn"
+                    data-translate="{{:: ctrlDomainTabZoneDns.constructor.getTranslateBtnKey()}}"
```

Comment:
Maybe we could have two button : an order button and on activate button and display it based on region ?

Reply:
Yes, i hesitate between two approach.
But now it seems more readable code.